assets/python/espresso_xml_get_qpt_irr.j2.py

```diff
@@ -8,6 +8,7 @@
 # ---------------------------------------------------------- #
 import json
 from xml.dom import minidom
+from __future__ import print_function
```

Comment:
does this run as python2?

Reply:
This file was still using the old Python2-style `print "spam"` syntax. I'm not sure if anything is still relying on it being Python2, so I went with the safer-route for back-compatibility